src/GovUk.Education.ExploreEducationStatistics.Content.Services/PublicationService.cs

```diff
@@ -246,6 +250,9 @@ private async Task<PublicationTreePublicationViewModel> BuildPublicationTreePubl
             Title = publication.Title,
             Slug = publication.Slug,
             IsSuperseded = await _publicationRepository.IsSuperseded(publication.Id),
+            SupersededById = publication.SupersededBy?.Id,
+            SupersededByTitle = publication.SupersededBy?.Title,
+            SupersededBySlug = publication.SupersededBy?.Slug,
```

Comment:
Also need additional testing for this too - for `PublicationService#GetPublicationTree`.

Reply:
Added `Get_PublicationTree_SupersededPublicationBuildsPublicationSupersededByViewModel` to`PublicationServiceTests.cs`. Like the other unit test added to this file, wasn't too sure about the naming (it seems a bit verbose)